RecoDataProducts/inc/HelixSeed.hh

```diff
@@ -11,6 +11,7 @@
 #include "Offline/RecoDataProducts/inc/RobustHelix.hh"
 #include "Offline/RecoDataProducts/inc/TimeCluster.hh"
 #include "Offline/RecoDataProducts/inc/TrkFitFlag.hh"
+#include "Offline/RecoDataProducts/inc/HelixRecoDir.hh"
```

Comment:
This new file is missing from the PR.
From the use downstream I see you persist slope and slope/error.  I suggest to persist slope and error (or better yet, variance): those are well defined, robust, and the ratio can be computed as a member function.